popup.js

```diff
@@ -83,11 +83,53 @@ document.addEventListener('DOMContentLoaded', function () { // this function  st
                 console.log("user has deleted the board manually.");
                 board_create_function();
             });
-
+        
+        // Populating Fields
         chrome.tabs.getSelected(null, function (tab) {
             document.getElementById('data_url').value = tab.url;
         });
 
+        function getFieldsFromDOM() {
+            var fields = []
+            
+            //Account for Companies which have no URL/Link
+            try {
+                var url = document.getElementsByClassName("jobs-details-top-card__company-url")[0].innerText.trim();
+            } catch {
+
+            }
+            if (url) {
+                fields.push(document.getElementsByClassName("jobs-details-top-card__company-url")[0].innerText.trim())
+            } else {
+                var childNodes = document.getElementsByClassName("jobs-details-top-card__company-info")[0].childNodes;
+                result = '';
+            
+                for (var i = 0; i < childNodes.length; i++) {
+                    if(childNodes[i].nodeType == 3) {
+                        result += childNodes[i].data;
+                    }
+                }
+
```

Comment:
Can you please remove blank lines in this function (wherever seems fit)? It makes the code look very far apart to the eyes!

Reply:
Yep not a problem! Will be done soon.